Sources/ArcGISToolkit/Components/Form/FormInputs/SwitchInput.swift

```diff
@@ -90,54 +83,23 @@ struct SwitchInput: View {
             .disabled(!inputModel.isEditable)
             .padding([.bottom], elementPadding)
             .onAppear {
-                if element.value.isEmpty {
+                if element.formattedValue.isEmpty {
                     fallbackToComboBox = true
                 } else {
-                    switchState = isOn
+                    isOn = input.onValue.name == inputModel.formattedValue
```

Comment:
You could compare codes here too, right? May be a bit faster (but not a big deal).

`input.onValue.code == inputModel.value`???

Or would you have to cast that to the correct code type?

Reply:
Correct, we'd need the cast which is why I'm depending on formattedValue to be the coded value's name.